thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/controller.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Controller for Alert Details Page: Tune Sensitivity Tab
+ * @module manage/alert/tune
+ * @exports manage/alert/tune
+ */
+import fetch from 'fetch';
+import Ember from 'ember';
+import _ from 'lodash';
+import { checkStatus, buildDateEod } from 'thirdeye-frontend/helpers/utils';
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+
+  /**
+   * Mapping anomaly table column names to corresponding prop keys
+   */
+  sortMap: {
+    name: 'name',
+    alert: 'alerts',
+    anomaly: 'data.totalAlerts',
+    user: 'data.userReportAnomaly',
+    responses: 'data.totalResponses',
+    resrate: 'data.responseRate',
+    precision: 'data.precision'
+  },
+
+  /**
+   * List of applications to render as rows, with filtering applied
+   */
+  applications: Ember.computed(
+    'perfDataByApplication',
+    'selectedSortMode',
+    'viewTotals',
+    'sortMap',
+    function() {
+      const sortMap = this.get('sortMap');
+      const selectedSortMode = this.get('selectedSortMode');
+      const sortMapChild = this.get('viewTotals') ? '.tot' : '.avg';
+      const keysWithChildren = ['anomaly', 'user', 'responses'];
+      let allApps = this.get('perfDataByApplication');
+      let fullSortKey = '';
+      let applySortType = true;
+
+      if (selectedSortMode) {
+        let [ sortKey, sortDir ] = selectedSortMode.split(':');
+        applySortType = keysWithChildren.includes(sortKey);
+        fullSortKey = applySortType ? sortMap[sortKey] + sortMapChild : sortMap[sortKey];
+
+        if (sortDir === 'up') {
+          allApps = _.sortBy(allApps, fullSortKey);
+        } else {
+          allApps = _.sortBy(allApps, fullSortKey).reverse();
+        }
+      }
+
+      return allApps;
+    }
+  ),
+
+  actions: {
+    /**
+     * Handle sorting for each sortable table column
+     * @param {String} sortKey  - stringified start date
+     */
+    toggleSortDirection(sortKey) {
+      const propName = 'sortColumn' + sortKey.capitalize() + 'Up' || '';
+
+      this.toggleProperty(propName);
+      if (this.get(propName)) {
```

Comment:
There are 2 different `propName` variables here declared in different scopes. Is that correct? If so, can we change the name to differentiate? It's a bit confusing.

Reply:
No, its the same var. We're constructing the property name.

Reply:
Then can we use `propName` on line 69 instead of `this.get('propName')`? Let me know if I'm misinterpreting this.